src/actions/actionTypes.js

```diff
@@ -0,0 +1,10 @@
+export const AJAX_CALL_ERROR = 'AJAX_CALL_ERROR';
+
+export const CLEAR_VERSION_HISTORY = 'CLEAR_VERSION_HISTORY';
+export const LOADED_VERSION_HISTORY = 'LOADED_VERSION_HISTORY';
+export const LOADED_VERSIONS = 'LOADED_VERSIONS';
+
+export const LOADING_VERSION_HISTORY = 'LOADING_VERSION_HISTORY';
```

Comment:
In yourturn we have the convention of `FETCH` prefix for ajax actions and `BEGIN_` prefix for ajax start, so in your case this would be `FETCH_VERSION_HISTORY` and `BEGIN_FETCH_VERSION_HISTORY`. Just fyi, maybe you like this naming pattern better.